modules/exploits/linux/local/cve_2021_4034_pwnkit_lpe_pkexec.rb

```diff
@@ -0,0 +1,90 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+  include Msf::Post::File
+  include Msf::Post::Linux::Priv
+  include Msf::Post::Linux::Kernel
+  include Msf::Post::Linux::System
+  include Msf::Post::Linux::Compile
+  include Msf::Exploit::EXE
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Local Privilege Escalation in polkits pkexec',
+        'Description' => %q{
+          A local privilege escalation vulnerability was found on polkit's pkexec utility.
+          The pkexec application is a setuid tool designed to allow unprivileged the
+          current version of pkexec doesn't handle the calling parameters count correctly
+          and ends trying to execute environment variables as commands.
+
+          An attacker can leverage this by crafting environment variables
+          in such a way it'll induce pkexec to execute arbitrary code. When successfully
+          executed the attack can cause a local privilege escalation given unprivileged
+          users administrative rights on the target machine.
```

Comment:
This is copied verbatim from the CVE description. The modules description should describe the vulnerability and methods used by the exploit. It should also describe tested/affected systems.